### PR TITLE
cfm: update to 0.4.4

### PR DESCRIPTION
**Pull request by WillEccles**
## Description of package
[cfm](https://github.com/willeccles/cfm) is a simple TUI file manager with no dependencies.

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/cfm/version [hunk 1]
```diff
@@ -1 +1 @@
-0.4.1 3
+0.4.4 4
```
Comment (dylanaraps) [inline community/cfm/version]: `0.4.4 1`. You only bump the second field when you change the package's files but there as no package release.

Comment (WillEccles) [reply community/cfm/version]: Could you elaborate on what this means? I'm not understanding properly. Fixed, however.

Comment (dylanaraps) [reply community/cfm/version]: - Software has an update -> First field (software version) is bumped.
- The KISS package files were changed (Possibly to change the `build` file or to fix a now invalid source as examples) -> Second field is bumped.

Comment (WillEccles) [reply community/cfm/version]: Okay, thanks! Will keep that in mind.

**Pull request merged**